packages/core/tsconfig.json

```diff
@@ -12,8 +12,14 @@
     "strict": true,
     "lib": ["es2015", "dom"],
     "target": "ES2015",
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+  },
+  "ts-node": {
+    "files": true,
+    "compilerOptions": {
+      "module": "commonjs",
+    },
   },
   "include": ["./src"],
-  "exclude": ["node_modules"]
+  "exclude": ["node_modules"],
```

Comment:
Why there are commas here? Have you ran prettier?

Reply:
@himself65 Yes!

`pnpm run format:write`

Reply:
we can setup `trailingComma: "none"` on another PR, was setting here but around 200 file changes

Reply:
Let's keep it as is. Makes sense.